tools/clang/test/DXC/disable_paq.hlsl

```diff
@@ -0,0 +1,18 @@
+// RUN: %dxc -T lib_6_7 -disable-payload-qualifiers %s 2>&1 | FileCheck %s
+// RUN: %dxc -T lib_6_8 -disable-payload-qualifiers %s 2>&1 | FileCheck %s
+
+// Ensures that -disable-payload-qualifiers works properly for SM6.6+ versions
+
+
+// CHECK: warning: payload access qualifiers ignored. These are only supported for lib_6_7+ targets and lib_6_6 with with the -enable-payload-qualifiers flag.
```

Comment:
Why we have warning for lib_6_7 and lib_6_8?

Reply:
In both of these cases, the payload qualifiers are being explicitly disabled, so we want a warning letting the user know that the annotations they are making are being ignored. The warning message doesn't mention that possibility, but it was already pretty long and I thought disabling them was both uncommon enough to skip mentioning and explicit enough that it wouldn't require mentioning.